snorkel/learning/gen_learning.py

```diff
@@ -159,6 +112,10 @@ class prior (in log scale)
         :param cardinality: number of possible classes; by default is inferred
             from the label matrix L
         :param timer: stopwatch for profiling, must implement start() and end()
+        :param scoped_categorical: If True, remap the values of a categorical
+        label matrix so that the support for each candidate is a contiguous
+        range, so that only relevant values are sampled during learning / 
```

Comment:
This needs to be clearer. That they're remapped to be contiguous is an implementation detail.